src/test/components/ActiveTabTimeline.test.js

```diff
@@ -349,6 +349,27 @@ describe('ActiveTabTimeline', function() {
         fireFullClick(getResourceTrackRow());
         expect(getFirstSelectedThreadIndex(getState())).toBe(threadIndex);
       });
+
+      it('can toggle a selected track by clicking the label', () => {
+        const {
+          getState,
+          getResourceFrameTrackLabel,
+          getResourceTrackRow,
+          threadIndex,
+        } = setup();
+        expect(getFirstSelectedThreadIndex(getState())).not.toBe(threadIndex);
+        expect(getResourceTrackRow().classList.contains('opened')).toBe(false);
+        fireFullClick(getResourceFrameTrackLabel());
+        expect(getFirstSelectedThreadIndex(getState())).toBe(threadIndex);
+        expect(getResourceTrackRow().classList.contains('opened')).toBe(true);
+      });
+
+      it('does not toggle a selected track by clicking other part of the track except label', () => {
+        const { getState, getResourceTrackRow, threadIndex } = setup();
+        expect(getFirstSelectedThreadIndex(getState())).not.toBe(threadIndex);
+        fireFullClick(getResourceTrackRow());
+        expect(getFirstSelectedThreadIndex(getState())).toBe(threadIndex);
```

Comment:
Can you also change the expect statements in this `it` block to both `expect(getResourceTrackRow().classList.contains('opened')).toBe(false);` instead? That way we can test to make sure we are not expanding them.
It should be both false before and after because it shouldn't collapse/expand the tracks.

Reply:
Ah, I missed that one. Let me add that and test again.